### ISSUE TITLE
Non-browser forms in MDI container do not redraw properly when moved

### ISSUE THREAD

**Issue by evconrad** [cefsharp/CefSharp#1945]
When there are multiple forms in a WinForms MDI container, when non-browser forms are moved over the form with an embedded browser, they do not repaint properly. When the browser form is again clicked on, the borders of the other forms get repainted, but not the main part of the forms.  It's possible that this is an issue with the CEF library.

![formtest](https://cloud.githubusercontent.com/assets/4349684/22983488/1b420df8-f357-11e6-87ce-a813ff160e32.png)

**Comment by amaitland** [comment]
Seems you've already opened an issue on the `CEF` issue tracker. If your experiencing `DotNetBrowser` and that's based on `CEF` as you claim (I've got no idea, never used it) then this would need to be fixed in `CEF`.

https://bitbucket.org/chromiumembedded/cef/issues/2086/non-browser-forms-in-mdi-container-do-not

Closing as `wontfix`, if it turns out that changes to `CefSharp` are required then this can be reopened.
